api/controller.py

```diff
@@ -249,10 +248,7 @@ def load_facets_from_request(self, *args, **kwargs):
             ):
                 return NO_SUCH_LANE.detailed(_("Lane does not exist"))
 
-        if (
-            isinstance(facets, BaseFacets)
-            and getattr(facets, "max_cache_age", None) is not None
-        ):
+        if isinstance(facets, BaseFacets):
             # A faceting object was loaded, and it tried to do something nonstandard
```

Comment:
This comment seems out of date, now that we aren't checking `max_cache_age` here. 

Can this whole `if` block just go away? It seems that it was mainly concerned with facet caching, which we aren't doing locally anymore.

Reply:
Yes, facets.max_cache_age is no longer used, will have to remove all occurences.